### DISCUSSION TITLE
PBCs and supercells

### DISCUSSION THREAD

**Discussion by terryfrankcombe** [mir-group/nequip#328] [Q&A] [answered]
If I train a model on a particular crystalline or amorphous cell, will the ASE calculator happily/correctly work if I ask it to give me an energy for an explicitly replicated supercell/different sized cell?  If so, are there any restrictions on this?

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @terryfrankcombe ,

Not sure I follow the question, but yes, our default hyperparameters are set up to be sane for giving correctly extensive/intensive predictions that can generalize in system size. For NequIP or Allegro in ASE you will be limited by what you can fit on one GPU; for Allegro you can scale to extremely large systems using `pair_allegro` in LAMMPS.
